project.clj

```diff
@@ -42,7 +42,7 @@
                    (println))}
 
   :dependencies [[org.clojure/clojure "1.10.0"]
-                 [yetibot/core "20190830.212304.0be0d9e"]
+                 [yetibot/core "20190902.141804.835fee7"]
```

Comment:
Guessing you `lein install`'d locally to test this out. Kinda cool that it works! This version scheme is only a few months old (I'll eventually move yetibot to the same datetime-and-sha-derived version scheme too).

Note we need to bump this to a publicly released version of `yetibot/core` before merging

Reply:
Yep.

I noticed `yetibot-codeclimate` depends on `yetibot-core`.
Are there any other artifacts that depend on core?

Reply:
`yetibot-stackstorm` and `yetibot-codeclimate` are both plugins that depend on `yetibot-core`, but neither of them have been maintained for awhile since I quit using them.

I think it's best to leave them outdated for now until some point where we get them working again.

Reply:
Can bump to `[yetibot/core "20190905.175835.fe16ae2"]`!

Reply:
👍 done